fastlane/Fastfile

```diff
@@ -59,13 +65,10 @@ platform :ios do
     sh("cd #{output_path}/#{xcframework_file_name}; find . -name '*.swiftinterface' -exec sed -i -e 's/MailchimpSDK\\.//g' {} \\;")
   end
 
-  desc "Create a release candidate for a given version"
-  lane :release do |args|
-    # Check for version argument
-    version = args[:version] || ENV['MAILCHIMP_SDK_VERSION'] || marketing_version
-    if version.nil?
-      UI.user_error! "Version not provided through argument, environment variable, or release branch name. Try running: `fastlane release version:MAJOR.MINOR.PATCH` or setting the environment variable named MAILCHIMP_SDK_VERSION, where MAJOR.MINOR.PATCH represents the version number."
-    end
+  desc "Create a release candidate"
+  lane :release do
+    # Always use MARKETING_VERSION for a Release Candidate. Use `set_version` to update MARKETING_VERSION
+    version = marketing_version
```

Comment:
Is it worth prompting here to ensure `set_version` is run beforehand?

Reply:
We use this for CI, so I avoided a prompt.

Reply:
Oh right, that makes sense 👍